src/runtime_src/core/common/api/xrt_error.cpp

```diff
@@ -284,3 +308,32 @@ xrtErrorGetString(xrtDeviceHandle, xrtErrorCode error, char* out, size_t len, si
   }
   return errno;
 }
+
+int
+xrtErrorGetJson(xrtDeviceHandle, xrtErrorCode error, char* out, size_t len, size_t* out_len)
```

Comment:
Please remove this method.  It isn't need if the code is calling the above mentioned method:
```
void
error_code_to_json(xrtErrorCode ecode, boost::property_tree:ptree & pt)
```

Reply:
done